advanced.md

```diff
@@ -45,7 +45,7 @@ Whenever you construct the `UwpFirmata` object yourself, you **must** call `star
 
 ```c#
 //member variables
-IStream bluetooth;
+IStream connection;
```

Comment:
Might be less confusing to call this bluetoothSerial - minimizes the number of visual concepts and cognitive load for the user (There are 3 - IStream, connection, Bluetooth, Serial)